base_jsonify/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Base JSONify",
     "summary": "Base module that provide the jsonify method on all models",
-    "version": "14.0.1.5.0",
```

Comment:
@oerp-odoo why changing from ??.0.1.5.0 to ??.0.1.**0**.0 i.e. removing the minor version number? Don't think that's correct.

Reply:
Actually thats how versions are bumped. If you bump major version, minor version is reset: https://semver.org/

Reply:
> Actually thats how versions are bumped. If you bump major version, minor version is reset: https://semver.org/

You're right. Strange I didn't notice yet - I always bumped major version and left the rest untouched. Still I consider this a bit strange. Reason being that major version bump is not really a bump at all - just 1:1 transfer to another version. This way we'll get a situation where 15.0.1.0.0 is functionally equal to 14.0.1.5.0. Or in case of 14.0 migration it went from 13.0.3.0.0 to 14.0.1.0.0.

Reply:
But thats OK, because major version is bumped:). It would look weird if other version numbers would be left, because it would look like more changes happened after migration, when it did not.